
geometry_msgs/Point goal_point
---
float32 elapsed_time
---
float32 distance_to_point

